File: Python/19-pattern-symmetry.py
def symmetry(n):
    for i in range(n):
        print("* " * (n - i), end="")
        print("  " * (2 * (i + 1)), end="")
        print("* " * (n - i))

    for i in range(n):
        print("* " * (i + 1), end="")
        print("  " * (2 * n), end="")
        print("* " * (i + 1))

n = int(input())
symmetry(n)

File: Java/19-pattern-symmetry.java
import java.util.Scanner;

public class Solution {
    public static void symmetry(int n) {
        // Write your code here
        for(int i = 0; i < n; i++) {
            for(int j = 0; j<n-i; j++) {
                System.out.print("* ");
            }
            
            for(int j = 0; j<2*(i+1); j++){
                System.out.print(" ");
            }

            for(int j = 0; j<n-i; j++) {
                System.out.print("* ");
            }

            System.out.println();
        }

        for(int i = 0; i<n; i++){
            for(int j = 0; j<i+1; j++){
                System.out.print("* ");
            }

            for(int j = 0; j<2*(n); j++){
                System.out.print(" ");
            }

            for(int j = 0; j<i+1; j++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        symmetry(n);
        scanner.close();
    }
}
